common/app/views/fragments/nav/editionPickerDropdown.scala.html

```diff
@@ -14,32 +26,16 @@
         id="edition-dropdown-menu"
         aria-hidden="true">
 
-        @Edition.all.map { edition =>
-            @if((edition != Edition(request))) {
-
-                <li class="dropdown-menu__item">
-                    <a data-link-name="nav2 : topbar : edition-picker: @edition.id"
-                        href="@LinkTo(s"/preference/edition/${edition.id.toLowerCase}")"
-                        class="dropdown-menu__title">
+        @editionListItem(Edition(request), true)
 
-                        <span class="u-h">switch to the </span>
-                        @edition.displayName
-                    </a>
-                </li>
-            }
-        }
+        @Edition.others(request).map(e => editionListItem(e))
     </ul>
 
     <label for="edition-picker-toggle"
             class="top-bar__item top-bar__item--dropdown js-edition-picker-trigger"
             data-link-name="nav2 : topnav : edition-picker: toggle">
 
-        <span class="u-h">Current Edition: </span>
```

Comment:
Can we keep this span? Its for screen readers.

Reply:
Good spot. I didn't mean to remove that.